lib/core/public/run-virtual-rule.js

```diff
@@ -37,10 +37,15 @@ function runVirtualRule(ruleId, vNode, options = {}) {
   // can avoid this call by forcing the rule to not exclude hidden
   // elements
   rule = Object.create(rule, { excludeHidden: { value: false } });
-
   const context = {
     initiator: true,
-    include: [vNode]
+    include: [vNode],
+    exclude: [],
+    frames: [],
+    page: false,
+    focusable: true,
+    size: {},
+    flatTree: []
```

Comment:
Spotted some missing props. Don't know if it can cause a problem, but I figured I'd fix it and add a test.